content/en/docs/components/training/overview.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "Overview"
+description = "An overview of Kubeflow Training Operator"
+weight = 5
++++
+
+Kubeflow Training Operator is a Kubernetes-native project for fine-tuning and scalable
+distributed training of machine learning (ML) models created with various ML frameworks such as
+PyTorch, Tensorflow, XGBoost, and others.
+
+Training Operator allows you to use Kubernetes workloads to effectively train your large models
+via Kubernetes Custom Resources APIs or using Training Operator Python SDK.
+
+Training Operator implements centralized Kubernetes controller to orchestrate distributed training jobs.
+
+## Architecture
+
+This diagram shows the major features of Training Operator and supported ML frameworks.
+
+<img src="/docs/components/training/images/training-operator-overview.drawio.png"
+  alt="Training Operator Overview"
+  class="mt-3 mb-3">
+
+## Custom Resources for ML Frameworks
+
+To perform distributed training Training Operator implements the following
+[Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/):
+
+| ML Framework | Custom Resource |
+| ------------ | --------------- |
+| PyTorch      | PyTorchJob      |
+| Tensorflow   | TFJob           |
+| Apache MXNet | MXJob           |
```

Comment:
Should we remove this mention going forward?

Reply:
@terrytangyuan Sure, removed it.